### ISSUE TITLE
[Demo] spec for image auth browser plugin

### ISSUE THREAD

**Issue by moven0831** [moven0831/ZK-IA#3]
# [Demo] spec for image auth browser plugin

### TLDR

Build a browser plugin that support in-browser proof generation for content creator and verification for content viewer

### Plugin's functionalities

- For Content Creators
    - Generate ZKPs for redacted images directly within the browser, using witness data from the local machine.
    - Upload the ZKPs to IPFS (or L1/L2s) to make them accessible to viewers.
    - Optionally disclose the metadata of the images, such as location and timestamp.
- For Content Viewers
    - Verify the ZKPs embedded in the article through in-browser processes.
    - Display the verification results alongside the images.
    - Access and view any revealed metadata associated with the images.

![ZK Study Group  Redactable signature scheme for image](https://github.com/moven0831/ZK-IA/assets/60170228/496accb9-cd1d-4c7f-bdaf-eb0ab4ff9fa0)
